lightning-data-persister/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "lightning-data-persister"
+version = "0.0.1"
+authors = ["Matt Corallo"]
```

Comment:
I dont think thats right :p

Reply:
Lol, updated. Fwiw Julian pointed out that we don't have a system for the "authors" section atm, so I went with a seeming default.